### DISCUSSION TITLE
Need help setting up

### DISCUSSION THREAD

**Discussion by ajbjc** [bucanero/apollo-psp#31] [Q&A] [answered]
What CFW is required Apollo as I'm using 6.61 Pro-C Infinity 2 (the permanent one). Also, how exactly do I set up and use Apollo? It says to just put the zip file in the PSP folder, but where specifically, and don't you usually place a regular folder in the seplugins folder and then add a line to a vsh file?

As for save data key dumping, it doesn't seem to work. SGDeemer only goes up to 6.60 and SGKeyDumper doesn't exactly work. It creates the .BIN file but in a different area. And it's just 16 bytes of nothing.

**Comment by bucanero** [comment, marked as answer]
The app should work on any CFW like ME, PRO, ARK. You install it as any other homebrew app, unzip and copy to the memory stick in the right folder `PSP/GAME`. You should end up with `ms0:/PSP/GAME/APOLLO/`

About plugins, if SGKeyDumper creates a 16-byte .BIN file, then it looks it is working properly.

**Reply by ajbjc** [reply]
Thanks, I'll try it, haven't done much with homebrew, mostly just installed a .CTF theme and installed another homebrew game.
